tests/packagedcode/data/package_summary/change-case-change-case-5.4.4.zip-extract/change-case-change-case-5.4.4/packages/change-case/README.md

```diff
@@ -0,0 +1,78 @@
+# Change Case
+
+> Transform a string between `camelCase`, `PascalCase`, `Capital Case`, `snake_case`, `kebab-case`, `CONSTANT_CASE` and others.
+
+## Installation
+
+```
+npm install change-case --save
+```
+
+## Usage
+
+```js
+import * as changeCase from "change-case";
+
+changeCase.camelCase("TEST_VALUE"); //=> "testValue"
+```
+
+Included case functions:
+
+| Method            | Result      |
+| ----------------- | ----------- |
+| `camelCase`       | `twoWords`  |
+| `capitalCase`     | `Two Words` |
+| `constantCase`    | `TWO_WORDS` |
+| `dotCase`         | `two.words` |
+| `kebabCase`       | `two-words` |
+| `noCase`          | `two words` |
+| `pascalCase`      | `TwoWords`  |
+| `pascalSnakeCase` | `Two_Words` |
+| `pathCase`        | `two/words` |
+| `sentenceCase`    | `Two words` |
+| `snakeCase`       | `two_words` |
+| `trainCase`       | `Two-Words` |
+
+All methods accept an `options` object as the second argument:
+
+- `delimiter?: string` The character to use between words. Default depends on method, e.g. `_` in snake case.
```

Comment:
Please remove unnecessary text from test files which are not required to test funcitonality.

Reply:
Resolved.